src/core/dom/layout/viewport-observer.js

```diff
@@ -42,23 +40,21 @@ const viewportCallbacks = new WeakMap();
  * @param {function(boolean)} viewportCallback
  */
 export function observeWithSharedInOb(element, viewportCallback) {
-  // There should never be two unique observers of the same element.
-  if (mode.isTest()) {
-    devAssert(
-      !viewportCallbacks.has(element) ||
-        viewportCallbacks.get(element) === viewportCallback
-    );
-  }
-
-  const win = toWin(element.ownerDocument.defaultView);
+  const win = toWin(element?.ownerDocument?.defaultView);
```

Comment:
`element` can't be nullish, and element's always have an `ownerDocument`.
```suggestion
  const win = toWin(element.ownerDocument.defaultView);
```